Times: 2
Path:
  count_num: 5
  point1:
    x: 2
    y: 2
  point2:
    x: 8
    y: 2
  point3:
    x: 8
    y: 8
  point4:
    x: 2
    y: 8
  point5:
    x: 5
    y: 5
